src/services/web-monitoring-db.ts

```diff
@@ -0,0 +1,156 @@
+/* tslint:disable interface-name */
+const defaultApiUrl = 'https://web-monitoring-db-staging.herokuapp.com/';
+
+export interface Version {
+    uuid: string;
+    page_uuid?: string;
```

Comment:
Gooooood question. I don’t recall and you’re right, that doesn’t really make any sense.